spec.html

```diff
@@ -14233,22 +14251,12 @@ <h1>Static Semantics: MV</h1>
         <h1>Static Semantics: NumericValue</h1>
         <emu-grammar>NumericLiteral :: DecimalLiteral</emu-grammar>
         <emu-alg>
-          1. Return the Number value that results from rounding the MV of |DecimalLiteral| as described below.
+          1. Return RoundStringMVResult(MV of |DecimalLiteral|).
         </emu-alg>
         <emu-grammar>NumericLiteral :: NonDecimalIntegerLiteral</emu-grammar>
         <emu-alg>
-          1. Return the Number value that results from rounding the MV of |NonDecimalIntegerLiteral| as described below.
+          1. Return RoundStringMVResult(MV of |NonDecimalIntegerLiteral|).
```

Comment:
```suggestion
          1. Return 𝔽(MV of |NonDecimalIntegerLiteral|).
```
As above, 20-digit rounding doesn't apply to non-decimal literals.

Reply:
Are you sure? It seems to apply in this case. Are you saying that it was a spec bug before?

Reply:
NonDecimalIntegerLiteral's MV *is* subject to "rounding ... as described below" but that amounts to just the standard "the Number value for x" (i.e., 𝔽). The 20-digit 2-option rounding is only invoked if the numeric literal is a DecimalLiteral.

Reply:
Oh I see, I missed this part:

> unless the literal is a |DecimalLiteral| and